browser/resources/settings/email_aliases_page/content/constant.ts

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) 2025 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+export const MAX_ALIASES = 5
```

Comment:
the ts seems like a strange place to define this, shouldn't it either be managed in the cpp code or at least be defined in the mojo file?

Reply:
I think a constant in the mojom would make most sense. I always forget that you can do that.

Reply:
I moved it to the mojom file